components/mpas-ocean/driver/ocn_comp_mct.F

```diff
@@ -2282,7 +2282,7 @@ subroutine ocn_import_mct(x2o_o, errorCode)!{{{
       call mpas_pool_get_field(forcingPool, 'peakWaveDirection', peakWaveDirectionField)
 
       call mpas_pool_get_field(forcingPool, 'landIceFreshwaterFlux', landIceFreshwaterFluxField)
-      call mpas_pool_get_field(forcingPool, 'landIceHeatFlux', landIceHeatFluxField)
+      call mpas_pool_get_array(forcingPool, 'heatFluxToLandIce', landIceHeatFluxField)
```

Comment:
It feels confusing to me to use `landIceHeatFluxField` to point to `heatFluxToLandIce`, since these variables were used to mean different things in the past.

If we want to rename them to something more meaningful, I would be in favor of that but that might be a huge can of worms.  Maybe better is just to rename the local variable:
`landIceHeatFlux` -->`heatFluxToLandIce`
`landIceHeatFluxField` -->`heatFluxToLandIceField`

Reply:
```suggestion
      call mpas_pool_get_array(forcingPool, 'heatFluxToLandIce', heatFluxToLandIceField)
```

Reply:
Got it, that makes total sense.